LICENSE.md

```diff
@@ -0,0 +1,64 @@
+# Instructional Material
+
+All instructional material is made available under the [Creative Commons 
+Attribution license][cc-by-human]. The following is a human-readable summary of 
+(and not a substitute for) the [full legal text of the CC BY 4.0 
+license][cc-by-legal].
+
+You are free to:
+
+* to **Share** --- copy and redistribute the material in any medium or format
+* to **Adapt** --- remix, transform, and build upon the material for any 
+  purpose, even commercially.
+
+The licensor cannot revoke these freedoms as long as you follow the license 
+terms.
+
+Under the following terms:
+
+* **Attribution** --- You must give appropriate credit, provide a [link to the 
+  license][cc-by-human], and indicate if changes were made. You may do so in 
+  any reasonable manner, but not in any way that suggests the licensor endorses 
+  you or your use.
+
+* **No additional restrictions** --- You may not apply legal terms or 
+  technological measures that legally restrict others from doing anything the 
+  license permits.
+
+Notices:
+
+* You do not have to comply with the license for elements of the material in 
+  the public domain or where your use is permitted by an applicable exception 
+  or limitation.
+* No warranties are given. The license may not give you all of the permissions 
+  necessary for your intended use. For example, other rights such as publicity, 
+  privacy, or moral rights may limit how you use the material.
+
+# Software
+
+Except where otherwise noted, the example programs and other software provided 
+by Software Carpentry are made available under the [OSI][osi]-approved [MIT 
+license][mit-license].
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of 
+this software and associated documentation files (the "Software"), to deal in 
+the Software without restriction, including without limitation the rights to 
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies 
+of the Software, and to permit persons to whom the Software is furnished to do 
+so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all 
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
```

Comment:
Shall one extend saying "SOFTWARE AND DOCUMENTATION"?

Reply:
OK, it would imply a related update of the section title and a couple of other places where the word "software" is mentioned.

Reply:
I think the point is that the text and pictures here are licensed under CC and the code is licensed under the MIT license. This is a common pattern as software licenses like MIT can't be applied to text and vice versa. I'd be -1 on changing any of the text of any of the licenses. Mostly because we aren't lawyers :)

Reply:
That's certainly a good reason to leave it as is! Fair enough @betatim.

Reply:
That is a copy of the MIT licence and doesn't apply to the instructional material which is under CC BY 4.0.

This is taken from the [starterkit-lessons repository](https://github.com/lhcb/starterkit-lessons/blob/master/LICENSE.md) and I think should be kept to cover any traces of the original software carpentry material.